introduction.md

```diff
@@ -0,0 +1,208 @@
+
+Some methods are explained below:
+
+- `.toUpperCase()` method: This method returns a string in all capital letters.
+
+```javascript
+console.log('hello'.toUpperCase()); // Prints 'HELLO'
+```
+ `.toUpperCase()` method is called on the string instance `'hello'`. The result is logged to the console. This method returns a string in all capital letters: `'HELLO'`.
+
+- `.startsWith()` method:
+
+  This method takes in an argument as checks if the string to which it is appended starts with that argument. If it starts with the argument then it prints `true` and if it does not start with the argument it prints `false`.
+
+```javascript
+console.log('Hey'.startsWith('H')); // Prints true
+```
+
+the `.startsWith()` method is called on the string instance `'Hey`'. This method also accepts the character `'H'` as an input, or argument, between the parentheses. Since the string `'Hey'` does start with the letter `'H'`, the method returns the boolean `true`.
+
+
+- `.trim()` method
+ The `.trim()` method is used to remove white spaces at the beginning and end of a string.
+
+ ```js
+console.log('    Remove whitespace   '.trim());
+ ```
+
+## Built-in Objects
+
+In addition to console, there are other [objects built into JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects). These objects have methods.
+For example, the built-in `Math` object in javascript is used to perform more complex mathematical operations than arithmetic. An example is calling the `.random()` method from the built-in Math object:
+
+```js
+console.log(Math.random()); // Prints a random number between 0 and 1
+```
+
+In the example above, we called the `.random()` method by appending the object name with the dot operator, the name of the method, and opening and closing parentheses. This method returns a random number between 0 and 1.
+
+To generate a random number between 0 and 50, we could multiply this result by 50, like below:
+
+```js
+Math.random() * 50;
+```
+The example above will likely evaluate to a decimal. To ensure an answer is a whole number, we can take advantage of another useful Math method called ```Math.floor()```.
+
+`Math.floor()` takes a decimal number, and rounds down to the nearest whole number. we can use `Math.floor()` to round down a random number like below:
+
+```js
+Math.floor(Math.random() * 50);
+```
+
+`Math.ceil()` takes a decimal number, and rounds up to the nearest whole number. we can use `Math.ceil()` to round up a random number like this:
+
+```js
+Math.ceil(Math.random() * 50);
+```
+
+- To see all of the properties and methods on the Math object, take a look at [the documentation here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math).
+
+- check the list of built-in string methods in the [JavaScript documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
```

Comment:
Add this to a new section. Maybe "Recommendation" to "Recommendation Resoucres" or "Further reading", ect

Add all the link you have learned from to write this article there.  And if you used anything there make sure you reference it properly.

Reply:
I'll add the section.